wherescape/connectors/hubspot/hubspot_wrapper.py

```diff
@@ -0,0 +1,98 @@
+import logging
+import hubspot
+
+from hubspot.crm.properties import ApiException
+from .send_data import (
+    send_company_object,
+    send_contact_object,
+    send_deal_object,
+)
+
+"""
+module that makes sure the data is prepared correctly depending on where and how it will be send.
+Also provides the properties depending on the object.
+"""
+
+
+class Hubspot:
+    def __init__(self, access_token):
+        self.access_token = access_token
+        self.client = hubspot.Client.create(access_token=access_token)
+
+    def get_company_properties(self):
+        """
+        this method returns all information of company properties
+        """
+        logging.info("getting properties of companies")
+        return self.get_object_properties("companies")
+
+    def get_deal_properties(self):
+        """
+        this method returns all information of deal properties
+        """
+        logging.info("getting properties of deals")
+        return self.get_object_properties("deals")
+
+    def get_contact_properties(self):
+        """
+        this method returns all information of contact properties
+        """
+        logging.info("getting properties of contacts")
+        return self.get_object_properties("contacts")
+
+    def send_company_patch(self, **args):
```

Comment:
```suggestion
    def send_company_patch(self, **properties):
```
args you can name change to anything you want, the more explaining variables the better!

To make it easier to understand what the input is of these functions I would simply to have a parameter that is the properties: list which is given in the function: def send_data(
    object_type: str, change_type: str, properties: list, hubspot_instance: Hubspot
):
hubspot_instance.send_company_patch(properties)
Then you can also skipline 47.